samples/LiveEffect/README.md

```diff
@@ -0,0 +1,25 @@
+LiveEffect Sample
+============
+
+This sample demonstrates how to loop back audio from input stream to output stream.
+
+Screenshots
+-----------
+
+![Screenshot](screenshot.png)
+
+
+### Stream Configurations
+- 48kHz
+- oboe::I16
+- steroe or mono
+
+
+### Caveats
+When first time starting audio devices, the stream may not be stable.
+The symtom is the strange callback pattern. This sample waits half a second
```

Comment:
spelling: symptom. Also, this could be worded better.

Reply:
done